app/src/test/java/me/hufman/androidautoidrive/notifications/StatusbarAppTest.kt

```diff
@@ -91,7 +91,7 @@ class StatusbarAppTest {
 	fun testNotificationCenterController() {
 		val mockServer = MockBMWRemotingServer()
 		IDriveConnection.mockRemotingServer = mockServer
-		val app = ID5StatusbarApp(iDriveConnectionStatus, securityAccess, carAppResources, graphicsHelpers)
+		val app = ID5StatusbarApp(iDriveConnectionStatus, securityAccess, carAppResources, mock(), graphicsHelpers)
 		val controller = app.statusbarController
 		val event = app.notificationEvent
 
```

Comment:
I think there's one more in testID5Popup that the [unit tests](https://github.com/BimmerGestalt/AAIdrive/actions/runs/10207416351/job/28242130977?pr=808) are complaining about:
```
file:///home/runner/work/AAIdrive/AAIdrive/app/src/test/java/me/hufman/androidautoidrive/notifications/StatusbarAppTest.kt:146:86 Type mismatch: inferred type is GraphicsHelpers but CarAppWidgetAssetResources was expected
```